erpnext/hr/doctype/employee/employee.js

```diff
@@ -97,3 +97,36 @@ frappe.ui.form.on('Employee',{
 	}
 });
 cur_frm.cscript = new erpnext.hr.EmployeeController({frm: cur_frm});
+
+frappe.tour['Employee'] = [
+	{
+		fieldname: "first_name",
+		title: "Enter Name",
+		description: __("Enter Employee Name."),
+	},
+	{
+		fieldname: "company",
+		title: "Company",
+		description: __("Select Company.")
+	},
+	{
+		fieldname: "gender",
+		title: "Gender",
+		description: __("select Gender.")
+	},
+	{
+		fieldname: "date_of_birth",
+		title: "Date of Birth",
+		description: __("Select Date of Birth.")
+	},
+	{
+		fieldname: "date_of_joining",
+		title: "Date of Joining",
+		description: __("Select Date of Joining.")
+	},
+	{
+		fieldname: "holiday_list",
+		title: "Holiday List",
+		description: __("Select Holiday list.")
+	},
```

Comment:
These look like self explanatory fields. The purpose of form tour isn't to explain each and every field, but to explain those that are confusing or need some documentation to get clarity about. A simple data field such as Name or even Company selection doesn't require explanation.
Unnecessary tour of fields could annoy a user and his experience. So try to limit this to fields that really need explanation